packages/storage-plus/src/indexed_map.rs

```diff
@@ -257,13 +258,13 @@ mod test {
 
         // match on proper age
         let proper = index_int(42);
```

Comment:
If you are using a named key, why use index_int (which hides all this stuff)?
We can just return `U32Key`

Reply:
All right.